app-modules/project/src/Filament/Resources/ProjectResource/Pages/ManageTasks.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Project\Filament\Resources\ProjectResource\Pages;
+
+use AdvisingApp\Project\Filament\Resources\ProjectResource;
+use App\Features\AssociateTasksWithProjectsFeature;
+use Filament\Resources\Pages\ManageRelatedRecords;
+use Filament\Tables\Actions\AssociateAction;
+use Filament\Tables\Actions\BulkActionGroup;
+use Filament\Tables\Actions\DissociateAction;
+use Filament\Tables\Actions\DissociateBulkAction;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+
+class ManageTasks extends ManageRelatedRecords
+{
+    protected static string $resource = ProjectResource::class;
+
+    protected static string $relationship = 'tasks';
+
+    public static function getNavigationLabel(): string
+    {
+        return 'Tasks';
+    }
+
+    public static function canAccess(array $parameters = []): bool
+    {
+        $user = auth()->user();
+
+        return AssociateTasksWithProjectsFeature::active() && $user->can(['project.*.view']) && parent::canAccess($parameters);
```

Comment:
I am not 100% sure here, but I think you can get rid of `$user->can(['project.*.view'])` as I think the parent Filament method should check if you can view the parent record of this `ManageRelatedRecords` page. Which should be validated by the access testing in your test.

If that doesn't work then it may make more sense to do `$user->can(['view', $this->getOwnerRecord())`